packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -90,10 +99,47 @@ export default class QuanticResultLink extends LightningElement {
     );
   };
 
+  checkResultType() {
```

Comment:
instead of this function that sets a state variable i think it would be simpler to just have a getter

```
get isSalesforceLink() {
  return this.result?.raw?.sfid !== undefined
}
```

Reply:
😄 I overthought that one